dicodile/data/images.py

```diff
@@ -1,15 +1,17 @@
 import os
+import pathlib
 import PIL
 import matplotlib.pyplot as plt
+from download import download
 
-
-DATA_DIR = os.environ.get("DATA_DIR", "../../data")
-
+from .home import DATA_HOME
 
 def get_mandril():
 
-    mandril = os.path.join(DATA_DIR,
-                           "images/standard_images/mandril_color.tif")
+    mandril_dir = pathlib.Path(DATA_HOME) / "images" / "standard_images"
```

Comment:
It is already a `Path` so I would say it is safe to do:
```suggestion
    mandril_dir = DATA_HOME / "images" / "standard_images"
```

Reply:
good catch, thanks!